Dockerfile

```diff
@@ -9,8 +9,15 @@ COPY build.rs /src/build.rs
 COPY migration /src/migration
 COPY src /src/src
 RUN cd app && npm ci
-RUN --mount=type=cache,target=/usr/local/cargo/registry --mount=type=cache,target=/src/target cargo build --profile release -p migration && cp /src/target/release/migration /migration
-RUN --mount=type=cache,target=/usr/local/cargo/registry --mount=type=cache,target=/src/target cargo build --profile release && cp /src/target/release/divviup_api_bin /divviup_api_bin
+ARG RUST_FEATURES=default
```

Comment:
I put the `ARG` way down here because AFAICT, if you vary a build arg, it invalidates all the image layers after the `ARG` directive. Putting this directive later in the Dockerfile makes it faster to build images with different `RUST_FEATURES` values.

Reply:
Extremely lightweight suggestion: If I were guessing the name of this without looking it up I'd probably guess `CARGO_FEATURES` or `FEATURES` instead of `RUST_FEATURES`